cmd/login/cmd.go

```diff
@@ -236,6 +236,13 @@ func run(cmd *cobra.Command, argv []string) {
 		(cfg.FedRAMP && token == "") ||
 		fedramp.IsGovRegion(arguments.GetRegion()) ||
 		config.IsEncryptedToken(token) {
+		// Display error to user if they attempt to log into govcloud without a region specified (fixes OCM-5718)
+		if !fedramp.IsGovRegion(arguments.GetRegion()) {
+			r.Reporter.Errorf("Must supply govcloud region when using flag '--govcloud'\n" +
```

Comment:
I'm personally thinking it'd be better if the message is more generic? Since there are different approaches to specifying whether a user wants to log into FedRAMP.

like "When logging into the FedRAMP environment, a recognized us-gov region needs to be specified. 
Example: --region us-gov-west-1"

Also, would you kindly add a unit test?
I think you can put the "or" statements into a separate function and try to emulate the error with mock arguments.

WDYT?

Reply:
I agree, will change the msg and add a unit test